src/cgame/cg_consolecmds.cpp

```diff
@@ -536,13 +550,9 @@ Cmd_Argc() / Cmd_Argv()
 bool ConsoleCommand()
 {
 	char buffer[BIG_INFO_STRING];
-	consoleCommand_t *cmd;
-
-	cmd = (consoleCommand_t*) bsearch( CG_Argv( 0 ), commands,
-	               ARRAY_LEN( commands ), sizeof( commands[ 0 ] ),
-	               cmdcmp );
 
-	if ( !cmd || !cmd->function )
+	const cg_cmd_t *cmd = std::lower_bound( std::begin( commands ), std::end( commands ), CG_Argv( 0 ) );
+	if ( ( cmd == std::end( commands ) && 0 == Q_stricmp( CG_Argv( 0 ), cmd->cmd ) ) || !cmd->function )
```

Comment:
`cmd == std::end( commands ) && 0 == Q_stricmp( CG_Argv( 0 ), cmd->cmd )`

This is wrong, it dereferences the end iterator

Reply:
should be fixed now